courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -327,14 +326,14 @@ Integer Overflows
    K : Short_Integer := Short_Integer'Last;
    ...
    K := K + 1;
-      
+
     2#0111_1111_1111_1111#  = (2**16)-1
 
-   +                    1
```

Comment:
Spacing is important - convert to a literal block?

Reply:
Indeed

Reply:
Fixed the linter ! :)